src/docfx/build/page/BuildPage.cs

```diff
@@ -198,12 +196,13 @@ private static (List<Error> errors, JObject model)
                 errors.Add(Errors.HeadingNotFound(file));
             }
 
+            var (_, inputMetadata) = context.MetadataProvider.GetMetadata(file);
```

Comment:
accumulate errors is

Reply:
it's calculated in build page, here we don't need do it.

Reply:
I know, but it is a good practice to not relying on behaviors elsewhere.